client/craco.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  style: {
+    postcss: {
+      plugins: [
+        require('tailwindcss'),
+        require('autoprefixer'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zerexei/zoop-note/pullRequest?prid=7527662)
```suggestion
        require("autoprefixer"),
```